### PR TITLE
Update README Badges

### PR DESCRIPTION
**Pull request by yinweisu**
*Issue #, if available:*

*Description of changes:*
* Update build badge to use github action badge
* Added platform tests badge

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,7 +7,8 @@
 ## AutoML for Image, Text, and Tabular Data
 
 [![Latest Release](https://img.shields.io/github/v/release/awslabs/autogluon)](https://github.com/awslabs/autogluon/releases)
-[![Build Status](https://ci.gluon.ai/view/all/job/autogluon/job/master/badge/icon)](https://ci.gluon.ai/view/all/job/autogluon/job/master/)
+[![Continuous Integration](https://github.com/awslabs/autogluon/actions/workflows/continuous_integration.yml/badge.svg)](https://github.com/awslabs/autogluon/actions/workflows/continuous_integration.yml)
+[![Platform Tests](https://github.com/awslabs/autogluon/actions/workflows/platform_tests-command.yml/badge.svg?event=schedule)](https://github.com/awslabs/autogluon/actions/workflows/platform_tests-command.yml)
```
Comment (gidler) [inline README.md:11]: Do you know what the `event=schedule` query parameter means on the badge? I loaded the badge with and without it and didn't notice any difference.

Comment (gidler) [reply README.md:11]: Sort of answering my own question, I found [this documentation](https://docs.github.com/en/actions/monitoring-and-troubleshooting-workflows/adding-a-workflow-status-badge#using-the-event-parameter)

Comment (yinweisu) [reply README.md:11]: Oh yea, for our platform tests we have different triggers. The schedule one is the one that runs every night. We want to show that.

Comment (gradientsky) [reply README.md:11]: This badge displays the status of workflow runs triggered by the `schedule` event. Open question here if that's the right event. [Examples](https://docs.github.com/en/actions/monitoring-and-troubleshooting-workflows/adding-a-workflow-status-badge#using-the-event-parameter) are using `push`.

Comment (yinweisu) [reply README.md:11]: @gradientsky it is the right event, as we want to display the status of the nightly tests not status of results from some manual trigger in a PR

**Review by gradientsky — APPROVED** [review]

**Comment by yinweisu** [comment]
Link commented by the bot is wrong and fixed. Will only take effect after merging

**Pull request merged**